interface/repository/pkmn_repository.go

```diff
@@ -0,0 +1,217 @@
+package repository
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strconv"
+
+	"github.com/jvillarreal-w/academy-go-q42021/common"
+	"github.com/jvillarreal-w/academy-go-q42021/domain/model"
+	"github.com/jvillarreal-w/academy-go-q42021/usecase/repository"
+)
+
+type pokemonRepository struct {
```

Comment:
I would highly suggest having the file path as an attribute here on your struct, that way when initializing a new Repository, you could provide any file path, which will make unit testing a lot easier since your not binding all of this logic to just one file. We don't want to make tests with the actual real file, but with test files, testing will come later but I thought it would be worth mentioning this now

Reply:
Added in a later commit, thank you.